ragna/deploy/_ui/central_view.py

```diff
@@ -110,6 +110,7 @@ def __init__(
         on_click_source_info_callback: Optional[Callable] = None,
         timestamp=None,
         show_timestamp=True,
+        toolbar_visible=True,  # hide the toolbar during streaming
```

Comment:
Should we make this

```suggestion
        toolbar_visible: Optional[bool] = None,  # hide the toolbar during streaming
```

and enforce a `None` value for all roles except `"assistant"`? I mean this is not user facing so not blocking in any way.

Reply:
The way the current code is designed, the `toolbar` is only used by the `assistant` role. I'll make this more obvious by renaming as `assistant_toolbar_visible`.